tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -143,3 +143,32 @@ def test_check_mds_cache_memory_limit(self):
         log.info(
             f"mds_cache_memory_limit is set with a value of {expected_mds_value_in_GB}GB"
         )
+
+    @bugzilla("2012930")
+    @post_ocs_upgrade
+    @pytest.mark.polarion_id("OCS-2739")
+    def test_noobaa_postgres_cm_post_ocs_upgrade(self):
+        """
+        Validate noobaa postgres configmap post OCS upgrade
+
+        """
+        cm_obj = OCP(
+            kind=constants.CONFIGMAP,
+            resource_name=constants.NOOBAA_POSTGRES_CONFIGMAP,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+        )
+        config_data = cm_obj.get().get("data").get("noobaa-postgres.conf")
+        config_data = config_data.split("\n")
+        config_data = config_data[slice(5, len(config_data))]
```

Comment:
`config_data[slice(5, len(config_data))]` could be written as `config_data[5:]`.

Reply:
Done, Thanks